Rest/Controller/ContentController.php

```diff
@@ -278,4 +292,53 @@ private function getRelation($content, $language)
 
         return false;
     }
+
+    /**
+     * Returns author of the content.
+     *
+     * @param Content $contentValue
+     * @param ContentType $contentType
+     *
+     * @return string
+     */
+    private function getAuthor(Content $contentValue, ContentType $contentType)
+    {
+        $author = $contentValue->getFieldValue($this->getFieldIdentifier('author', $contentType));
```

Comment:
We fetch value, only identifier is taken from config.
getFieldValue is is method from Content class.